coderd/database/queries/siteconfig.sql

```diff
@@ -131,3 +131,16 @@ SET value = CASE
     ELSE 'false'
 END
 WHERE site_configs.key = 'oauth2_github_default_eligible';
+
+-- name: UpsertWebpushVAPIDKeys :exec
+INSERT INTO site_configs (key, value)
+VALUES
+    ('webpush_vapid_public_key', @vapid_public_key :: text),
+    ('webpush_vapid_private_key', @vapid_private_key :: text)
+ON CONFLICT (key)
+DO UPDATE SET value = EXCLUDED.value WHERE site_configs.key = EXCLUDED.key;
+
+-- name: GetWebpushVAPIDKeys :one
+SELECT
+    COALESCE((SELECT value FROM site_configs WHERE key = 'webpush_vapid_public_key'), '') :: text AS vapid_public_key,
+    COALESCE((SELECT value FROM site_configs WHERE key = 'webpush_vapid_private_key'), '') :: text AS vapid_private_key;
```

Comment:
to be confirmed: this won't be a problem for multi-org setup? having a single keypair

Reply:
Not sure yet, but I think it's a deployment-level thing.